scripts/tls/openssl.cnf

```diff
@@ -29,6 +29,7 @@ DNS.1 = *.intel.com
 DNS.2 = k8s
 DNS.3 = kubernetes.default
 IP.1  = 127.0.0.1
+IP.2  = 10.10.0.2
```

Comment:
This will be the address of the inframanager running on ACC. As it is not static, I think we should document it in the readme for the user to update it before he proceeds with make gen-certs.

Reply:
No, it's not static. This is just for guidance. We need this covered in readme. I wanted to put this here to take care of the default config. Use can add as many addresses as it wants for "-r <IP ADDR> configured value.

Reply:
I think such things should also be mentioned in comments in the script